// tailwindcss/lib/plugins/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "preflight", {
  enumerable: true,
  get: function () {
    return _preflight.default;
  }
});
Object.defineProperty(exports, "container", {
  enumerable: true,
  get: function () {
    return _container.default;
  }
});
Object.defineProperty(exports, "accessibility", {
  enumerable: true,
  get: function () {
    return _accessibility.default;
  }
});
Object.defineProperty(exports, "pointerEvents", {
  enumerable: true,
  get: function () {
    return _pointerEvents.default;
  }
});
Object.defineProperty(exports, "visibility", {
  enumerable: true,
  get: function () {
    return _visibility.default;
  }
});
Object.defineProperty(exports, "position", {
  enumerable: true,
  get: function () {
    return _position.default;
  }
});
Object.defineProperty(exports, "inset", {
  enumerable: true,
  get: function () {
    return _inset.default;
  }
});
Object.defineProperty(exports, "isolation", {
  enumerable: true,
  get: function () {
    return _isolation.default;
  }
});
Object.defineProperty(exports, "zIndex", {
  enumerable: true,
  get: function () {
    return _zIndex.default;
  }
});
Object.defineProperty(exports, "order", {
  enumerable: true,
  get: function () {
    return _order.default;
  }
});
Object.defineProperty(exports, "gridColumn", {
  enumerable: true,
  get: function () {
    return _gridColumn.default;
  }
});
Object.defineProperty(exports, "gridColumnStart", {
  enumerable: true,
  get: function () {
    return _gridColumnStart.default;
  }
});
Object.defineProperty(exports, "gridColumnEnd", {
  enumerable: true,
  get: function () {
    return _gridColumnEnd.default;
  }
});
Object.defineProperty(exports, "gridRow", {
  enumerable: true,
  get: function () {
    return _gridRow.default;
  }
});
Object.defineProperty(exports, "gridRowStart", {
  enumerable: true,
  get: function () {
    return _gridRowStart.default;
  }
});
Object.defineProperty(exports, "gridRowEnd", {
  enumerable: true,
  get: function () {
    return _gridRowEnd.default;
  }
});
Object.defineProperty(exports, "float", {
  enumerable: true,
  get: function () {
    return _float.default;
  }
});
Object.defineProperty(exports, "clear", {
  enumerable: true,
  get: function () {
    return _clear.default;
  }
});
Object.defineProperty(exports, "margin", {
  enumerable: true,
  get: function () {
    return _margin.default;
  }
});
Object.defineProperty(exports, "boxSizing", {
  enumerable: true,
  get: function () {
    return _boxSizing.default;
  }
});
Object.defineProperty(exports, "display", {
  enumerable: true,
  get: function () {
    return _display.default;
  }
});
Object.defineProperty(exports, "height", {
  enumerable: true,
  get: function () {
    return _height.default;
  }
});
Object.defineProperty(exports, "maxHeight", {
  enumerable: true,
  get: function () {
    return _maxHeight.default;
  }
});
Object.defineProperty(exports, "minHeight", {
  enumerable: true,
  get: function () {
    return _minHeight.default;
  }
});
Object.defineProperty(exports, "width", {
  enumerable: true,
  get: function () {
    return _width.default;
  }
});
Object.defineProperty(exports, "minWidth", {
  enumerable: true,
  get: function () {
    return _minWidth.default;
  }
});
Object.defineProperty(exports, "maxWidth", {
  enumerable: true,
  get: function () {
    return _maxWidth.default;
  }
});
Object.defineProperty(exports, "flex", {
  enumerable: true,
  get: function () {
    return _flex.default;
  }
});
Object.defineProperty(exports, "flexShrink", {
  enumerable: true,
  get: function () {
    return _flexShrink.default;
  }
});
Object.defineProperty(exports, "flexGrow", {
  enumerable: true,
  get: function () {
    return _flexGrow.default;
  }
});
Object.defineProperty(exports, "tableLayout", {
  enumerable: true,
  get: function () {
    return _tableLayout.default;
  }
});
Object.defineProperty(exports, "borderCollapse", {
  enumerable: true,
  get: function () {
    return _borderCollapse.default;
  }
});
Object.defineProperty(exports, "transformOrigin", {
  enumerable: true,
  get: function () {
    return _transformOrigin.default;
  }
});
Object.defineProperty(exports, "translate", {
  enumerable: true,
  get: function () {
    return _translate.default;
  }
});
Object.defineProperty(exports, "rotate", {
  enumerable: true,
  get: function () {
    return _rotate.default;
  }
});
Object.defineProperty(exports, "skew", {
  enumerable: true,
  get: function () {
    return _skew.default;
  }
});
Object.defineProperty(exports, "scale", {
  enumerable: true,
  get: function () {
    return _scale.default;
  }
});
Object.defineProperty(exports, "transform", {
  enumerable: true,
  get: function () {
    return _transform.default;
  }
});
Object.defineProperty(exports, "animation", {
  enumerable: true,
  get: function () {
    return _animation.default;
  }
});
Object.defineProperty(exports, "cursor", {
  enumerable: true,
  get: function () {
    return _cursor.default;
  }
});
Object.defineProperty(exports, "userSelect", {
  enumerable: true,
  get: function () {
    return _userSelect.default;
  }
});
Object.defineProperty(exports, "resize", {
  enumerable: true,
  get: function () {
    return _resize.default;
  }
});
Object.defineProperty(exports, "listStylePosition", {
  enumerable: true,
  get: function () {
    return _listStylePosition.default;
  }
});
Object.defineProperty(exports, "listStyleType", {
  enumerable: true,
  get: function () {
    return _listStyleType.default;
  }
});
Object.defineProperty(exports, "appearance", {
  enumerable: true,
  get: function () {
    return _appearance.default;
  }
});
Object.defineProperty(exports, "gridAutoColumns", {
  enumerable: true,
  get: function () {
    return _gridAutoColumns.default;
  }
});
Object.defineProperty(exports, "gridAutoFlow", {
  enumerable: true,
  get: function () {
    return _gridAutoFlow.default;
  }
});
Object.defineProperty(exports, "gridAutoRows", {
  enumerable: true,
  get: function () {
    return _gridAutoRows.default;
  }
});
Object.defineProperty(exports, "gridTemplateColumns", {
  enumerable: true,
  get: function () {
    return _gridTemplateColumns.default;
  }
});
Object.defineProperty(exports, "gridTemplateRows", {
  enumerable: true,
  get: function () {
    return _gridTemplateRows.default;
  }
});
Object.defineProperty(exports, "flexDirection", {
  enumerable: true,
  get: function () {
    return _flexDirection.default;
  }
});
Object.defineProperty(exports, "flexWrap", {
  enumerable: true,
  get: function () {
    return _flexWrap.default;
  }
});
Object.defineProperty(exports, "placeContent", {
  enumerable: true,
  get: function () {
    return _placeContent.default;
  }
});
Object.defineProperty(exports, "placeItems", {
  enumerable: true,
  get: function () {
    return _placeItems.default;
  }
});
Object.defineProperty(exports, "alignContent", {
  enumerable: true,
  get: function () {
    return _alignContent.default;
  }
});
Object.defineProperty(exports, "alignItems", {
  enumerable: true,
  get: function () {
    return _alignItems.default;
  }
});
Object.defineProperty(exports, "justifyContent", {
  enumerable: true,
  get: function () {
    return _justifyContent.default;
  }
});
Object.defineProperty(exports, "justifyItems", {
  enumerable: true,
  get: function () {
    return _justifyItems.default;
  }
});
Object.defineProperty(exports, "gap", {
  enumerable: true,
  get: function () {
    return _gap.default;
  }
});
Object.defineProperty(exports, "space", {
  enumerable: true,
  get: function () {
    return _space.default;
  }
});
Object.defineProperty(exports, "divideWidth", {
  enumerable: true,
  get: function () {
    return _divideWidth.default;
  }
});
Object.defineProperty(exports, "divideStyle", {
  enumerable: true,
  get: function () {
    return _divideStyle.default;
  }
});
Object.defineProperty(exports, "divideColor", {
  enumerable: true,
  get: function () {
    return _divideColor.default;
  }
});
Object.defineProperty(exports, "divideOpacity", {
  enumerable: true,
  get: function () {
    return _divideOpacity.default;
  }
});
Object.defineProperty(exports, "placeSelf", {
  enumerable: true,
  get: function () {
    return _placeSelf.default;
  }
});
Object.defineProperty(exports, "alignSelf", {
  enumerable: true,
  get: function () {
    return _alignSelf.default;
  }
});
Object.defineProperty(exports, "justifySelf", {
  enumerable: true,
  get: function () {
    return _justifySelf.default;
  }
});
Object.defineProperty(exports, "overflow", {
  enumerable: true,
  get: function () {
    return _overflow.default;
  }
});
Object.defineProperty(exports, "overscrollBehavior", {
  enumerable: true,
  get: function () {
    return _overscrollBehavior.default;
  }
});
Object.defineProperty(exports, "textOverflow", {
  enumerable: true,
  get: function () {
    return _textOverflow.default;
  }
});
Object.defineProperty(exports, "whitespace", {
  enumerable: true,
  get: function () {
    return _whitespace.default;
  }
});
Object.defineProperty(exports, "wordBreak", {
  enumerable: true,
  get: function () {
    return _wordBreak.default;
  }
});
Object.defineProperty(exports, "borderRadius", {
  enumerable: true,
  get: function () {
    return _borderRadius.default;
  }
});
Object.defineProperty(exports, "borderWidth", {
  enumerable: true,
  get: function () {
    return _borderWidth.default;
  }
});
Object.defineProperty(exports, "borderStyle", {
  enumerable: true,
  get: function () {
    return _borderStyle.default;
  }
});
Object.defineProperty(exports, "borderColor", {
  enumerable: true,
  get: function () {
    return _borderColor.default;
  }
});
Object.defineProperty(exports, "borderOpacity", {
  enumerable: true,
  get: function () {
    return _borderOpacity.default;
  }
});
Object.defineProperty(exports, "backgroundColor", {
  enumerable: true,
  get: function () {
    return _backgroundColor.default;
  }
});
Object.defineProperty(exports, "backgroundOpacity", {
  enumerable: true,
  get: function () {
    return _backgroundOpacity.default;
  }
});
Object.defineProperty(exports, "backgroundImage", {
  enumerable: true,
  get: function () {
    return _backgroundImage.default;
  }
});
Object.defineProperty(exports, "gradientColorStops", {
  enumerable: true,
  get: function () {
    return _gradientColorStops.default;
  }
});
Object.defineProperty(exports, "boxDecorationBreak", {
  enumerable: true,
  get: function () {
    return _boxDecorationBreak.default;
  }
});
Object.defineProperty(exports, "backgroundSize", {
  enumerable: true,
  get: function () {
    return _backgroundSize.default;
  }
});
Object.defineProperty(exports, "backgroundAttachment", {
  enumerable: true,
  get: function () {
    return _backgroundAttachment.default;
  }
});
Object.defineProperty(exports, "backgroundClip", {
  enumerable: true,
  get: function () {
    return _backgroundClip.default;
  }
});
Object.defineProperty(exports, "backgroundPosition", {
  enumerable: true,
  get: function () {
    return _backgroundPosition.default;
  }
});
Object.defineProperty(exports, "backgroundRepeat", {
  enumerable: true,
  get: function () {
    return _backgroundRepeat.default;
  }
});
Object.defineProperty(exports, "backgroundOrigin", {
  enumerable: true,
  get: function () {
    return _backgroundOrigin.default;
  }
});
Object.defineProperty(exports, "fill", {
  enumerable: true,
  get: function () {
    return _fill.default;
  }
});
Object.defineProperty(exports, "stroke", {
  enumerable: true,
  get: function () {
    return _stroke.default;
  }
});
Object.defineProperty(exports, "strokeWidth", {
  enumerable: true,
  get: function () {
    return _strokeWidth.default;
  }
});
Object.defineProperty(exports, "objectFit", {
  enumerable: true,
  get: function () {
    return _objectFit.default;
  }
});
Object.defineProperty(exports, "objectPosition", {
  enumerable: true,
  get: function () {
    return _objectPosition.default;
  }
});
Object.defineProperty(exports, "padding", {
  enumerable: true,
  get: function () {
    return _padding.default;
  }
});
Object.defineProperty(exports, "textAlign", {
  enumerable: true,
  get: function () {
    return _textAlign.default;
  }
});
Object.defineProperty(exports, "verticalAlign", {
  enumerable: true,
  get: function () {
    return _verticalAlign.default;
  }
});
Object.defineProperty(exports, "fontFamily", {
  enumerable: true,
  get: function () {
    return _fontFamily.default;
  }
});
Object.defineProperty(exports, "fontSize", {
  enumerable: true,
  get: function () {
    return _fontSize.default;
  }
});
Object.defineProperty(exports, "fontWeight", {
  enumerable: true,
  get: function () {
    return _fontWeight.default;
  }
});
Object.defineProperty(exports, "textTransform", {
  enumerable: true,
  get: function () {
    return _textTransform.default;
  }
});
Object.defineProperty(exports, "fontStyle", {
  enumerable: true,
  get: function () {
    return _fontStyle.default;
  }
});
Object.defineProperty(exports, "fontVariantNumeric", {
  enumerable: true,
  get: function () {
    return _fontVariantNumeric.default;
  }
});
Object.defineProperty(exports, "lineHeight", {
  enumerable: true,
  get: function () {
    return _lineHeight.default;
  }
});
Object.defineProperty(exports, "letterSpacing", {
  enumerable: true,
  get: function () {
    return _letterSpacing.default;
  }
});
Object.defineProperty(exports, "textColor", {
  enumerable: true,
  get: function () {
    return _textColor.default;
  }
});
Object.defineProperty(exports, "textOpacity", {
  enumerable: true,
  get: function () {
    return _textOpacity.default;
  }
});
Object.defineProperty(exports, "textDecoration", {
  enumerable: true,
  get: function () {
    return _textDecoration.default;
  }
});
Object.defineProperty(exports, "fontSmoothing", {
  enumerable: true,
  get: function () {
    return _fontSmoothing.default;
  }
});
Object.defineProperty(exports, "placeholderColor", {
  enumerable: true,
  get: function () {
    return _placeholderColor.default;
  }
});
Object.defineProperty(exports, "placeholderOpacity", {
  enumerable: true,
  get: function () {
    return _placeholderOpacity.default;
  }
});
Object.defineProperty(exports, "caretColor", {
  enumerable: true,
  get: function () {
    return _caretColor.default;
  }
});
Object.defineProperty(exports, "opacity", {
  enumerable: true,
  get: function () {
    return _opacity.default;
  }
});
Object.defineProperty(exports, "backgroundBlendMode", {
  enumerable: true,
  get: function () {
    return _backgroundBlendMode.default;
  }
});
Object.defineProperty(exports, "mixBlendMode", {
  enumerable: true,
  get: function () {
    return _mixBlendMode.default;
  }
});
Object.defineProperty(exports, "boxShadow", {
  enumerable: true,
  get: function () {
    return _boxShadow.default;
  }
});
Object.defineProperty(exports, "outline", {
  enumerable: true,
  get: function () {
    return _outline.default;
  }
});
Object.defineProperty(exports, "ringWidth", {
  enumerable: true,
  get: function () {
    return _ringWidth.default;
  }
});
Object.defineProperty(exports, "ringColor", {
  enumerable: true,
  get: function () {
    return _ringColor.default;
  }
});
Object.defineProperty(exports, "ringOpacity", {
  enumerable: true,
  get: function () {
    return _ringOpacity.default;
  }
});
Object.defineProperty(exports, "ringOffsetWidth", {
  enumerable: true,
  get: function () {
    return _ringOffsetWidth.default;
  }
});
Object.defineProperty(exports, "ringOffsetColor", {
  enumerable: true,
  get: function () {
    return _ringOffsetColor.default;
  }
});
Object.defineProperty(exports, "blur", {
  enumerable: true,
  get: function () {
    return _blur.default;
  }
});
Object.defineProperty(exports, "brightness", {
  enumerable: true,
  get: function () {
    return _brightness.default;
  }
});
Object.defineProperty(exports, "contrast", {
  enumerable: true,
  get: function () {
    return _contrast.default;
  }
});
Object.defineProperty(exports, "dropShadow", {
  enumerable: true,
  get: function () {
    return _dropShadow.default;
  }
});
Object.defineProperty(exports, "grayscale", {
  enumerable: true,
  get: function () {
    return _grayscale.default;
  }
});
Object.defineProperty(exports, "hueRotate", {
  enumerable: true,
  get: function () {
    return _hueRotate.default;
  }
});
Object.defineProperty(exports, "invert", {
  enumerable: true,
  get: function () {
    return _invert.default;
  }
});
Object.defineProperty(exports, "saturate", {
  enumerable: true,
  get: function () {
    return _saturate.default;
  }
});
Object.defineProperty(exports, "sepia", {
  enumerable: true,
  get: function () {
    return _sepia.default;
  }
});
Object.defineProperty(exports, "filter", {
  enumerable: true,
  get: function () {
    return _filter.default;
  }
});
Object.defineProperty(exports, "backdropBlur", {
  enumerable: true,
  get: function () {
    return _backdropBlur.default;
  }
});
Object.defineProperty(exports, "backdropBrightness", {
  enumerable: true,
  get: function () {
    return _backdropBrightness.default;
  }
});
Object.defineProperty(exports, "backdropContrast", {
  enumerable: true,
  get: function () {
    return _backdropContrast.default;
  }
});
Object.defineProperty(exports, "backdropGrayscale", {
  enumerable: true,
  get: function () {
    return _backdropGrayscale.default;
  }
});
Object.defineProperty(exports, "backdropHueRotate", {
  enumerable: true,
  get: function () {
    return _backdropHueRotate.default;
  }
});
Object.defineProperty(exports, "backdropInvert", {
  enumerable: true,
  get: function () {
    return _backdropInvert.default;
  }
});
Object.defineProperty(exports, "backdropOpacity", {
  enumerable: true,
  get: function () {
    return _backdropOpacity.default;
  }
});
Object.defineProperty(exports, "backdropSaturate", {
  enumerable: true,
  get: function () {
    return _backdropSaturate.default;
  }
});
Object.defineProperty(exports, "backdropSepia", {
  enumerable: true,
  get: function () {
    return _backdropSepia.default;
  }
});
Object.defineProperty(exports, "backdropFilter", {
  enumerable: true,
  get: function () {
    return _backdropFilter.default;
  }
});
Object.defineProperty(exports, "transitionProperty", {
  enumerable: true,
  get: function () {
    return _transitionProperty.default;
  }
});
Object.defineProperty(exports, "transitionDelay", {
  enumerable: true,
  get: function () {
    return _transitionDelay.default;
  }
});
Object.defineProperty(exports, "transitionDuration", {
  enumerable: true,
  get: function () {
    return _transitionDuration.default;
  }
});
Object.defineProperty(exports, "transitionTimingFunction", {
  enumerable: true,
  get: function () {
    return _transitionTimingFunction.default;
  }
});
Object.defineProperty(exports, "content", {
  enumerable: true,
  get: function () {
    return _content.default;
  }
});

var _preflight = _interopRequireDefault(require("tailwindcss/lib/plugins/preflight"));

var _container = _interopRequireDefault(require("tailwindcss/lib/plugins/container"));

var _accessibility = _interopRequireDefault(require("tailwindcss/lib/plugins/accessibility"));

var _pointerEvents = _interopRequireDefault(require("tailwindcss/lib/plugins/pointerEvents"));

var _visibility = _interopRequireDefault(require("tailwindcss/lib/plugins/visibility"));

var _position = _interopRequireDefault(require("tailwindcss/lib/plugins/position"));

var _inset = _interopRequireDefault(require("tailwindcss/lib/plugins/inset"));

var _isolation = _interopRequireDefault(require("tailwindcss/lib/plugins/isolation"));

var _zIndex = _interopRequireDefault(require("tailwindcss/lib/plugins/zIndex"));

var _order = _interopRequireDefault(require("tailwindcss/lib/plugins/order"));

var _gridColumn = _interopRequireDefault(require("tailwindcss/lib/plugins/gridColumn"));

var _gridColumnStart = _interopRequireDefault(require("tailwindcss/lib/plugins/gridColumnStart"));

var _gridColumnEnd = _interopRequireDefault(require("tailwindcss/lib/plugins/gridColumnEnd"));

var _gridRow = _interopRequireDefault(require("tailwindcss/lib/plugins/gridRow"));

var _gridRowStart = _interopRequireDefault(require("tailwindcss/lib/plugins/gridRowStart"));

var _gridRowEnd = _interopRequireDefault(require("tailwindcss/lib/plugins/gridRowEnd"));

var _float = _interopRequireDefault(require("tailwindcss/lib/plugins/float"));

var _clear = _interopRequireDefault(require("tailwindcss/lib/plugins/clear"));

var _margin = _interopRequireDefault(require("tailwindcss/lib/plugins/margin"));

var _boxSizing = _interopRequireDefault(require("tailwindcss/lib/plugins/boxSizing"));

var _display = _interopRequireDefault(require("tailwindcss/lib/plugins/display"));

var _height = _interopRequireDefault(require("tailwindcss/lib/plugins/height"));

var _maxHeight = _interopRequireDefault(require("tailwindcss/lib/plugins/maxHeight"));

var _minHeight = _interopRequireDefault(require("tailwindcss/lib/plugins/minHeight"));

var _width = _interopRequireDefault(require("tailwindcss/lib/plugins/width"));

var _minWidth = _interopRequireDefault(require("tailwindcss/lib/plugins/minWidth"));

var _maxWidth = _interopRequireDefault(require("tailwindcss/lib/plugins/maxWidth"));

var _flex = _interopRequireDefault(require("tailwindcss/lib/plugins/flex"));

var _flexShrink = _interopRequireDefault(require("tailwindcss/lib/plugins/flexShrink"));

var _flexGrow = _interopRequireDefault(require("tailwindcss/lib/plugins/flexGrow"));

var _tableLayout = _interopRequireDefault(require("tailwindcss/lib/plugins/tableLayout"));

var _borderCollapse = _interopRequireDefault(require("tailwindcss/lib/plugins/borderCollapse"));

var _transformOrigin = _interopRequireDefault(require("tailwindcss/lib/plugins/transformOrigin"));

var _translate = _interopRequireDefault(require("tailwindcss/lib/plugins/translate"));

var _rotate = _interopRequireDefault(require("tailwindcss/lib/plugins/rotate"));

var _skew = _interopRequireDefault(require("tailwindcss/lib/plugins/skew"));

var _scale = _interopRequireDefault(require("tailwindcss/lib/plugins/scale"));

var _transform = _interopRequireDefault(require("tailwindcss/lib/plugins/transform"));

var _animation = _interopRequireDefault(require("tailwindcss/lib/plugins/animation"));

var _cursor = _interopRequireDefault(require("tailwindcss/lib/plugins/cursor"));

var _userSelect = _interopRequireDefault(require("tailwindcss/lib/plugins/userSelect"));

var _resize = _interopRequireDefault(require("tailwindcss/lib/plugins/resize"));

var _listStylePosition = _interopRequireDefault(require("tailwindcss/lib/plugins/listStylePosition"));

var _listStyleType = _interopRequireDefault(require("tailwindcss/lib/plugins/listStyleType"));

var _appearance = _interopRequireDefault(require("tailwindcss/lib/plugins/appearance"));

var _gridAutoColumns = _interopRequireDefault(require("tailwindcss/lib/plugins/gridAutoColumns"));

var _gridAutoFlow = _interopRequireDefault(require("tailwindcss/lib/plugins/gridAutoFlow"));

var _gridAutoRows = _interopRequireDefault(require("tailwindcss/lib/plugins/gridAutoRows"));

var _gridTemplateColumns = _interopRequireDefault(require("tailwindcss/lib/plugins/gridTemplateColumns"));

var _gridTemplateRows = _interopRequireDefault(require("tailwindcss/lib/plugins/gridTemplateRows"));

var _flexDirection = _interopRequireDefault(require("tailwindcss/lib/plugins/flexDirection"));

var _flexWrap = _interopRequireDefault(require("tailwindcss/lib/plugins/flexWrap"));

var _placeContent = _interopRequireDefault(require("tailwindcss/lib/plugins/placeContent"));

var _placeItems = _interopRequireDefault(require("tailwindcss/lib/plugins/placeItems"));

var _alignContent = _interopRequireDefault(require("tailwindcss/lib/plugins/alignContent"));

var _alignItems = _interopRequireDefault(require("tailwindcss/lib/plugins/alignItems"));

var _justifyContent = _interopRequireDefault(require("tailwindcss/lib/plugins/justifyContent"));

var _justifyItems = _interopRequireDefault(require("tailwindcss/lib/plugins/justifyItems"));

var _gap = _interopRequireDefault(require("tailwindcss/lib/plugins/gap"));

var _space = _interopRequireDefault(require("tailwindcss/lib/plugins/space"));

var _divideWidth = _interopRequireDefault(require("tailwindcss/lib/plugins/divideWidth"));

var _divideStyle = _interopRequireDefault(require("tailwindcss/lib/plugins/divideStyle"));

var _divideColor = _interopRequireDefault(require("tailwindcss/lib/plugins/divideColor"));

var _divideOpacity = _interopRequireDefault(require("tailwindcss/lib/plugins/divideOpacity"));

var _placeSelf = _interopRequireDefault(require("tailwindcss/lib/plugins/placeSelf"));

var _alignSelf = _interopRequireDefault(require("tailwindcss/lib/plugins/alignSelf"));

var _justifySelf = _interopRequireDefault(require("tailwindcss/lib/plugins/justifySelf"));

var _overflow = _interopRequireDefault(require("tailwindcss/lib/plugins/overflow"));

var _overscrollBehavior = _interopRequireDefault(require("tailwindcss/lib/plugins/overscrollBehavior"));

var _textOverflow = _interopRequireDefault(require("tailwindcss/lib/plugins/textOverflow"));

var _whitespace = _interopRequireDefault(require("tailwindcss/lib/plugins/whitespace"));

var _wordBreak = _interopRequireDefault(require("tailwindcss/lib/plugins/wordBreak"));

var _borderRadius = _interopRequireDefault(require("tailwindcss/lib/plugins/borderRadius"));

var _borderWidth = _interopRequireDefault(require("tailwindcss/lib/plugins/borderWidth"));

var _borderStyle = _interopRequireDefault(require("tailwindcss/lib/plugins/borderStyle"));

var _borderColor = _interopRequireDefault(require("tailwindcss/lib/plugins/borderColor"));

var _borderOpacity = _interopRequireDefault(require("tailwindcss/lib/plugins/borderOpacity"));

var _backgroundColor = _interopRequireDefault(require("tailwindcss/lib/plugins/backgroundColor"));

var _backgroundOpacity = _interopRequireDefault(require("tailwindcss/lib/plugins/backgroundOpacity"));

var _backgroundImage = _interopRequireDefault(require("tailwindcss/lib/plugins/backgroundImage"));

var _gradientColorStops = _interopRequireDefault(require("tailwindcss/lib/plugins/gradientColorStops"));

var _boxDecorationBreak = _interopRequireDefault(require("tailwindcss/lib/plugins/boxDecorationBreak"));

var _backgroundSize = _interopRequireDefault(require("tailwindcss/lib/plugins/backgroundSize"));

var _backgroundAttachment = _interopRequireDefault(require("tailwindcss/lib/plugins/backgroundAttachment"));

var _backgroundClip = _interopRequireDefault(require("tailwindcss/lib/plugins/backgroundClip"));

var _backgroundPosition = _interopRequireDefault(require("tailwindcss/lib/plugins/backgroundPosition"));

var _backgroundRepeat = _interopRequireDefault(require("tailwindcss/lib/plugins/backgroundRepeat"));

var _backgroundOrigin = _interopRequireDefault(require("tailwindcss/lib/plugins/backgroundOrigin"));

var _fill = _interopRequireDefault(require("tailwindcss/lib/plugins/fill"));

var _stroke = _interopRequireDefault(require("tailwindcss/lib/plugins/stroke"));

var _strokeWidth = _interopRequireDefault(require("tailwindcss/lib/plugins/strokeWidth"));

var _objectFit = _interopRequireDefault(require("tailwindcss/lib/plugins/objectFit"));

var _objectPosition = _interopRequireDefault(require("tailwindcss/lib/plugins/objectPosition"));

var _padding = _interopRequireDefault(require("tailwindcss/lib/plugins/padding"));

var _textAlign = _interopRequireDefault(require("tailwindcss/lib/plugins/textAlign"));

var _verticalAlign = _interopRequireDefault(require("tailwindcss/lib/plugins/verticalAlign"));

var _fontFamily = _interopRequireDefault(require("tailwindcss/lib/plugins/fontFamily"));

var _fontSize = _interopRequireDefault(require("tailwindcss/lib/plugins/fontSize"));

var _fontWeight = _interopRequireDefault(require("tailwindcss/lib/plugins/fontWeight"));

var _textTransform = _interopRequireDefault(require("tailwindcss/lib/plugins/textTransform"));

var _fontStyle = _interopRequireDefault(require("tailwindcss/lib/plugins/fontStyle"));

var _fontVariantNumeric = _interopRequireDefault(require("tailwindcss/lib/plugins/fontVariantNumeric"));

var _lineHeight = _interopRequireDefault(require("tailwindcss/lib/plugins/lineHeight"));

var _letterSpacing = _interopRequireDefault(require("tailwindcss/lib/plugins/letterSpacing"));

var _textColor = _interopRequireDefault(require("tailwindcss/lib/plugins/textColor"));

var _textOpacity = _interopRequireDefault(require("tailwindcss/lib/plugins/textOpacity"));

var _textDecoration = _interopRequireDefault(require("tailwindcss/lib/plugins/textDecoration"));

var _fontSmoothing = _interopRequireDefault(require("tailwindcss/lib/plugins/fontSmoothing"));

var _placeholderColor = _interopRequireDefault(require("tailwindcss/lib/plugins/placeholderColor"));

var _placeholderOpacity = _interopRequireDefault(require("tailwindcss/lib/plugins/placeholderOpacity"));

var _caretColor = _interopRequireDefault(require("tailwindcss/lib/plugins/caretColor"));

var _opacity = _interopRequireDefault(require("tailwindcss/lib/plugins/opacity"));

var _backgroundBlendMode = _interopRequireDefault(require("tailwindcss/lib/plugins/backgroundBlendMode"));

var _mixBlendMode = _interopRequireDefault(require("tailwindcss/lib/plugins/mixBlendMode"));

var _boxShadow = _interopRequireDefault(require("tailwindcss/lib/plugins/boxShadow"));

var _outline = _interopRequireDefault(require("tailwindcss/lib/plugins/outline"));

var _ringWidth = _interopRequireDefault(require("tailwindcss/lib/plugins/ringWidth"));

var _ringColor = _interopRequireDefault(require("tailwindcss/lib/plugins/ringColor"));

var _ringOpacity = _interopRequireDefault(require("tailwindcss/lib/plugins/ringOpacity"));

var _ringOffsetWidth = _interopRequireDefault(require("tailwindcss/lib/plugins/ringOffsetWidth"));

var _ringOffsetColor = _interopRequireDefault(require("tailwindcss/lib/plugins/ringOffsetColor"));

var _blur = _interopRequireDefault(require("tailwindcss/lib/plugins/blur"));

var _brightness = _interopRequireDefault(require("tailwindcss/lib/plugins/brightness"));

var _contrast = _interopRequireDefault(require("tailwindcss/lib/plugins/contrast"));

var _dropShadow = _interopRequireDefault(require("tailwindcss/lib/plugins/dropShadow"));

var _grayscale = _interopRequireDefault(require("tailwindcss/lib/plugins/grayscale"));

var _hueRotate = _interopRequireDefault(require("tailwindcss/lib/plugins/hueRotate"));

var _invert = _interopRequireDefault(require("tailwindcss/lib/plugins/invert"));

var _saturate = _interopRequireDefault(require("tailwindcss/lib/plugins/saturate"));

var _sepia = _interopRequireDefault(require("tailwindcss/lib/plugins/sepia"));

var _filter = _interopRequireDefault(require("tailwindcss/lib/plugins/filter"));

var _backdropBlur = _interopRequireDefault(require("tailwindcss/lib/plugins/backdropBlur"));

var _backdropBrightness = _interopRequireDefault(require("tailwindcss/lib/plugins/backdropBrightness"));

var _backdropContrast = _interopRequireDefault(require("tailwindcss/lib/plugins/backdropContrast"));

var _backdropGrayscale = _interopRequireDefault(require("tailwindcss/lib/plugins/backdropGrayscale"));

var _backdropHueRotate = _interopRequireDefault(require("tailwindcss/lib/plugins/backdropHueRotate"));

var _backdropInvert = _interopRequireDefault(require("tailwindcss/lib/plugins/backdropInvert"));

var _backdropOpacity = _interopRequireDefault(require("tailwindcss/lib/plugins/backdropOpacity"));

var _backdropSaturate = _interopRequireDefault(require("tailwindcss/lib/plugins/backdropSaturate"));

var _backdropSepia = _interopRequireDefault(require("tailwindcss/lib/plugins/backdropSepia"));

var _backdropFilter = _interopRequireDefault(require("tailwindcss/lib/plugins/backdropFilter"));

var _transitionProperty = _interopRequireDefault(require("tailwindcss/lib/plugins/transitionProperty"));

var _transitionDelay = _interopRequireDefault(require("tailwindcss/lib/plugins/transitionDelay"));

var _transitionDuration = _interopRequireDefault(require("tailwindcss/lib/plugins/transitionDuration"));

var _transitionTimingFunction = _interopRequireDefault(require("tailwindcss/lib/plugins/transitionTimingFunction"));

var _content = _interopRequireDefault(require("tailwindcss/lib/plugins/content"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
